app/src/pages/inside/projectSettingsPageContainer/content/notifications/ruleGroup/ruleGroup.jsx

```diff
@@ -0,0 +1,302 @@
+    },
+  ];
+
+  return (
+    <div className={cx('rule-section')}>
+      <Layout description={''}>
+        <div className={cx('rule-section-header')}>
+          <FieldElement
+            className={cx('fieldElement')}
+            withoutProvider
+            description={formatMessage(messages.typeDescription, { type: pluginName })}
+          >
+            <div className={cx('toggle')}>
+              <Toggle
+                disabled={isReadOnly}
+                value // poxel
+                onChange={(e) => toggleNotificationsEnabled(e.target.checked)}
+                dataAutomationId="enabledToggle"
+              >
+                <span className={cx('name-wrapper')}>
+                  <i className={cx('capitalized')}> {pluginName}</i>
+                </span>
+              </Toggle>
+            </div>
+          </FieldElement>
+          {!enabled ? (
+            <div className={cx('disabled-plugin')}>
+              <p>
+                <span className={cx('capitalized')}>{pluginName} </span>
+                {formatMessage(messages.disabledPlugin, { pluginName })}
+              </p>
+              <RuleItemDisabledTooltip
+                className={cx('info-tooltip')}
+                tooltipContent={formatMessage(messages.disabledContactInfo)}
+              >
+                <i className={cx('icon', 'about-icon')}>{Parser(AboutIcon)}</i>
+              </RuleItemDisabledTooltip>
+            </div>
+          ) : (
+            ''
+          )}
+        </div>
+
+        {enabled && (
+          <div className={cx('notifications-container')}>
+            {typedRules?.length > 0 ? (
+              <div className={cx('contentItems')}>
+                <RuleList
+                  disabled={isReadOnly}
+                  data={typedRules.map((rule) => ({
+                    name: rule.ruleName,
+                    ...rule,
+                  }))}
+                  actions={actions}
+                  onToggle={onToggleHandler}
+                  ruleItemContent={NotificationRuleContent}
+                  handleRuleItemClick={handleRuleItemClick}
+                  dataAutomationId="notificationsRulesList"
+                />
+                <Button
+                  customClassName={cx('add-rule')}
+                  onClick={onAdd}
+                  variant={'text'}
+                  startIcon={plusIcon}
+                >
+                  {formatMessage(messages.addRule)}
+                </Button>
+              </div>
+            ) : (
+              <EmptyRuleState ruleName={pluginName} onCreateClick={onAdd} />
+            )}
+          </div>
+        )}
+      </Layout>
+    </div>
+  );
+};
+const ruleShape = PropTypes.shape({
+  id: PropTypes.number.isRequired,
+  ruleName: PropTypes.string.isRequired,
+  recipients: PropTypes.array.isRequired,
+  sendCase: PropTypes.string.isRequired,
+  launchNames: PropTypes.array.isRequired,
+  attributes: PropTypes.array.isRequired,
```

Comment:
As I know at least the attributes are not required.
Pls check the limitations and/or discuss with BA if needed.